docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -746,8 +752,11 @@ a careful read of census quality assessment documentation.
 
 A master SSN list will be another important part of this, and perhaps
 the largest of these files, including name, address, DOB, and SSN.
-(Or should this list be a linkage output, derived from annually
-simulated tax return documents?)
+This list should be a linkage output, derived from annually simulated
+tax return documents, which include accurate SSN values year over year
```

Comment:
I see that elsewhere in the document you say SSN should "eventually" be missing/duplicated. When that changes, this will have to include some strategy for how people without SSNs file taxes.

Reply:
Good call. There is something called an "Individual Taxpayer Identification Number" that SSA creates for people who file taxes and do not have a SSN: https://www.irs.gov/individuals/international-taxpayers/taxpayer-identification-numbers-tin

There are also people who do not file taxes that I hope to eventually include in the simulation and some of them will not have an SSN or an ITIN.